policies/cla.yml

```diff
@@ -111,6 +111,11 @@ configuration:
          - podman-desktop-bot
          - liurunliang-bot
          - playwrightmachine
+         - openpublishbuild
```

Comment:
If you want to update the bots list for MicrosoftDocs org then this change has to be done in here https://github.com/MicrosoftDocs/.github/blob/main/policies/cla.yml